java/clients/src/main/java/sleeper/clients/admin/AdminConfigStore.java

```diff
@@ -182,6 +193,30 @@ public CouldNotLoadTableProperties(String instanceId, String tableName, Throwabl
         }
     }
 
+    public CompactionJobStatusStore loadCompactionJobStatusStore(String instanceId) {
+        InstanceProperties instanceProperties = loadInstanceProperties(instanceId);
+        return DynamoDBCompactionJobStatusStore.from(dynamoDB, instanceProperties);
+    }
+
+    public CompactionTaskStatusStore loadCompactionTaskStatusStore(String instanceId) {
+        InstanceProperties instanceProperties = loadInstanceProperties(instanceId);
+        return DynamoDBCompactionTaskStatusStore.from(dynamoDB, instanceProperties);
+    }
+
+    public IngestJobStatusStore loadIngestJobStatusStore(String instanceId) {
+        InstanceProperties instanceProperties = loadInstanceProperties(instanceId);
+        return DynamoDBIngestJobStatusStore.from(dynamoDB, instanceProperties);
+    }
+
+    public IngestTaskStatusStore loadIngestTaskStatusStore(String instanceId) {
+        InstanceProperties instanceProperties = loadInstanceProperties(instanceId);
+        return DynamoDBIngestTaskStatusStore.from(dynamoDB, instanceProperties);
+    }
+
+    public AmazonSQS getSqsClient() {
+        return sqs;
+    }
```

Comment:
This can be handled in https://github.com/gchq/sleeper/issues/711, https://github.com/gchq/sleeper/issues/712, https://github.com/gchq/sleeper/issues/709.